src/components/NFTcard/nftcard.tsx

```diff
@@ -48,7 +51,10 @@ const expiredAuction=useIsExpiredAuction(auctionNFT)
       </div>
       <Actions>
 
+
+
         <ButtonWishlist nftId={cardContent.id} type='NFTCard' disabled={expiredAuction} />
```

Comment:
if I already have this auction on my wishlist, I no longer can remove it after the auction is ended. @abdelaziz013  Kindly correct me if I'm worong

Reply:
for now, i can remove it , but this beyond the scope of this PR.

Reply:
it's not beyond the scope it will create another issue in my opinion. I think if we just leave it without disabling it if the auction is ended. @MoAminUIUX  kindly confirm the best UX for this case.